CMakeLists.txt

```diff
@@ -1220,7 +1220,9 @@ else()
         set(RC_CONFIG resources/soft_oal.rc)
     endif()
 
-    add_library(${IMPL_TARGET} SHARED ${OPENAL_OBJS} ${ALC_OBJS} ${RC_CONFIG})
+    # !important: for osx framework public header works, the headers must be in project
+    set(TARGET_PUBLIC_HEADERS include/AL/al.h;include/AL/alc.h;include/AL/alext.h;include/AL/efx.h)
```

Comment:
These semicolons don't look proper. If they're supposed to be separate elements in a list, spaces would be better to use.

Reply:
Yeah, please see the latest commit, fixed

Reply:
I have checked again, space is works since write as:
```cmake
PUBLIC_HEADER "${TARGET_PUBLIC_HEADERS}"
```
I will change later.

Reply:
modified